rest/route/service_test.go

```diff
@@ -1054,6 +1061,49 @@ func TestTaskResetPrepare(t *testing.T) {
 
 			So(err, ShouldResemble, expectedErr)
 		})
+
+		projCtx.ProjectRef = &serviceModel.ProjectRef{
+			Id:         "project_1",
+			Identifier: "project_identifier",
+		}
+
+		failedOnlyTest := func(failedOnly bool) {
+			projCtx.Task = &testTask
+			goodBod := &struct {
+				Failed_Only bool
```

Comment:
I would prefer we write this like this: https://github.com/evergreen-ci/evergreen/blob/e01cd853d9e70d17bef8e16ae80890b37eee0a86/rest/route/admin_test.go#L387-L391 (i.e. use the tags rather than giving the temporary variable name a camelcase format) or better yet https://github.com/evergreen-ci/evergreen/blob/e01cd853d9e70d17bef8e16ae80890b37eee0a86/rest/route/distro_test.go#L95 (since this is how a user would actually write it).
Also side note that camelcase is lowercase: https://www.golangprograms.com/golang-convert-string-into-snake-case.html